akka-stream-typed/src/main/scala/akka/stream/package.scala

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream
+
+package object typed {
+  type ActorMaterializer = akka.stream.ActorMaterializer
+}
```

Comment:
Hmmm I see... somewhat ugly for java since the alias won't be able to be used... should we mirror the interface in typed and do a wrapper perhaps?